gui-wm/wayfire/wayfire-0.7.5.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
I don't have time to check it today, most probably in 2 days or so, but can you consider bumping EAPI to 8?

Reply:
Thanks for your time, commit with EAPI 8 pushed.

Note: I am not feeling very comfortable in EAPI details yet though. The docs and cheatsheets I found are good but seem to be addressed to long-term devs. I'd need something from ground up.